skill-tree /src/main/java/com/RDS/skilltree/model/Skill.java

```diff
@@ -0,0 +1,13 @@
+package com.RDS.skilltree.model;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+@Entity
+public final class Skill { // TODO: to be updated according to the database
+    @Id
+    private Long id;
```

Comment:
Why is ID a long and not a UUID/ULID. There is no @GeneratedValue annotation as well

Reply:
// TODO: to be updated according to the database
I have added the comment that we will not be using this model and update it according to the requirements